contracts/yield-source/ATokenYieldSource.sol

```diff
@@ -156,7 +156,7 @@ contract ATokenYieldSource is ERC20Upgradeable, IProtocolYieldSource, AssetManag
   /// @param shares Amount of shares
   /// @return Number of tokens
   function _sharesToToken(uint256 shares) internal view returns (uint256) {
-    uint256 tokens = 0;
+    uint256 tokens;
```

Comment:
How much is the gas saving?  Assignment is more readable.

Reply:
If we look at `supplyTokenTo` that calls `_sharesToToken`, on the main branch it consumes on average 157,718 units of gas and on this branch, it consumes on average `157,718` units of gas, so it doesn't seem to change a thing. 😅
I agree that assignment is more readable.